brew_view/controllers/request_api.py

```diff
@@ -41,6 +54,27 @@ def get(self, request_id):
 
         self.write(self.parser.serialize_request(req, to_string=False))
 
+    def _update_completed_request_metrics(self, request):
+        latency = (
+            (datetime.datetime.utcnow() - request.updated_at).total_seconds() * 1000
```

Comment:
Does this give us actual millisecond precision?

Reply:
Yes